.travis.yml

```diff
@@ -241,7 +246,8 @@ script:
   - ./autogen.sh
   - CONFIGURE_FLAGS="$ASCIIDOC_OPTIONS $COVERAGE_OPTIONS $HARDENING_OPTIONS $MODULES_OPTIONS $NSS_OPTIONS $OPENSSL_OPTIONS $RUST_OPTIONS --enable-fatal-warnings --disable-silent-rules"
   - echo "Configure flags are $CONFIGURE_FLAGS CC=\"$CC $C_DIALECT_OPTIONS\""
-  - ./configure $CONFIGURE_FLAGS CC="$CC $C_DIALECT_OPTIONS"
+  - if [[ "$ALL_BUGS_ARE_FATAL" == "" ]]; then ./configure $CONFIGURE_FLAGS CC="$CC $C_DIALECT_OPTIONS"; fi
```

Comment:
I think it would make more sense to use  CONFIGURE_FLAGS or CFLAGS or something for this option, and have only one single `./configure` line.  If we had a separate "if" condition for every way that we can run ./configure, we would quickly hit a combinatorial explosion.

Reply:
Fixed in 6b3f4bd28febaefc696b03ba32bcc6adefb1b383.